### PR TITLE
#164829376 Adds functionality to calculate the reading time of an article

### PR DESCRIPTION
**Pull request by Raywire**
**What does this PR do?**
Adds a feature where a user is able to see the time it takes to read an article.

**Description of the task to be completed**

1. When a GET request to retrieve an article is made, the returned payload now contains the reading time for an article.

2. When a PUT request to update an article is made, and the article body is updated, the new reading time is updated.

**How should this be manually tested?**

- Open your terminal window and clone the repo,  and navigate to the root directory.
- Fetch all the branches and checkout into this feature's branch using the commands below
```bash
git fetch --all
git checkout ft-add-article-readtime-164829376
```
- Create a virtual environment and then activate the environment by as shown below
```python
virtualenv venv
source venv/bin/activate
```
- Install all the required dependencies using pip by running 
```python
pip install -r requirements.txt
```
- Make the new migrations and migrate using the commands below
```python
python manage.py makemigrations
python manage.py migrate
```
- Run the Django server to serve up the app by typing 
```python
python manage.py runserver
```
- Using postman, send a request with the data described in the format below.
- On the Auth tab select a Bearer Token as the type and on the Token field add the token retrieved from the login endpoint.

**Create an article (POST: localhost/api/articles/)**
Notice the read time is now returned in the payload.
Screenshot 2019-04-08 at 18 32 10

**Update an article (PUT: localhost/api/articles/authors-haven-test-four-74c2f6013c1c)**
Notice the read time is now updated in the payload.
Screenshot 2019-04-08 at 18 28 08

***Notes***
In case you had created previous articles, you may need to set a default reading time when prompted, then upon updating the article the actual reading time will be added.

**Relevant PT stories**
https://www.pivotaltracker.com/story/show/164829376

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -30,24 +30,30 @@ mccabe==0.6.1
...
 python-dateutil==2.8.0
 python-decouple==3.1
 python-dotenv==0.10.1
 python-http-client==3.1.0
+python3-openid==3.1.0
 pytz==2018.9
+readtime==1.1.1
 requests==2.21.0
+requests-oauthlib==1.2.0
+selenium==3.141.0
```
Comment (WNjihia) [inline requirements.txt:50]: selenium is a testing tool. I think it should be in dev requirements

**Review by WNjihia — CHANGES_REQUESTED** [review]

**Comment by WNjihia** [comment]
Looks good

File: requirements.txt (hunk 1 above)
Comment (Raywire) [reply requirements.txt:50]: Let me move it to the dev-requirements.

**Review by LarryKarani — DISMISSED** [review]
I have tested it locally everything works as described in the PR. Good work on the PR it's detailed

**Review by WNjihia — APPROVED** [review]

**Review by ipaullly — APPROVED** [review]
the functionality was implemented with minimum lines of code, that is admirable

**Pull request merged**